packages/help-menu/src/help-menu-templates.js

```diff
@@ -3,19 +3,33 @@ const helpMenuHeadContent = `
     <prm-icon icon-type="svg" svg-icon-set="navigation" icon-definition="ic_arrow_back_24px"
               aria-label="return to help content list" ></prm-icon>
   </md-button>
-  <h2>
+  <h1>
     <strong ng-if="helpMenuTitle">{{helpMenuTitle}}</strong>
     <strong ng-hide="helpMenuTitle">Search Help</strong>
     <span ng-hide="!entry"> - {{entry.title}}</span>
-  </h2>`;
+  </h1>`;
 
 const helpMenuMainContent = `
   <div ng-if="entry" id="search-help-menu-content" tabindex="-1">
     <br />
     <p ng-if="!entry.template"><em>{{entry.description}}</em></p>
     <div ng-bind-html="entry.template"></div>
   </div>
-  <ul ng-hide="entry" style="list-style: none; width: 100%; padding-left: 0px;">
+  <h2 ng-hide="entry || !helpContentUpdates">{{updatesLabel}}</h2>
+  <ul ng-hide="entry || !helpContentUpdates" style="list-style: none; width: 100%; padding-left: 0px;">
```

Comment:
this is to avoid recursions, we don't want to show the headings if there isn't a list beneath it